sw/device/lib/crypto/drivers/entropy.c

```diff
@@ -246,7 +246,7 @@ static const entropy_complex_config_t
                 .edn0 =
                     {
                         .base_address = kBaseEdn0,
-                        .reseed_interval = 32,
+                        .reseed_interval = 0x10000000,
```

Comment:
This seems very aggressive and I am not sure if it would be better to leave it as is.

With the current generate length of 32 128-bit block and the max reseed interval for keymgr, keymgr will request 64 bits every 65535 clock cycles. At 100 MHz, that's 0.65 ms. So 1 Generate will last for 32 * 128 / 64 * 0.65ms = 41.6 ms. And 32 Generates will last for 1.3 seconds. This is already longer than some tests run for.

I think with 0x1000000 we are already in the years range.

Reply:
This sounds good to me, thanks @moidx .

Reply:
BTW, I have now created a PR to change the hardware https://github.com/lowRISC/opentitan/pull/22951
But I still need to run the full regression and there might be some TLT fallout. So we shouldn't wait with this PR here.